systems/includes/rhea-compute-nodes.txt

```diff
@@ -14,16 +14,16 @@ Rhea contains 521 compute nodes separated into two partitions:
 
```

Comment:
I don't really see any reason to use a separate file for this section. Let's just go ahead and inline this in the right spot in the user guide.

Reply:
it is now in-line